exercises/practice/hamming/.meta/proof.ci.js

```diff
@@ -2,16 +2,8 @@ export const compute = (strand1, strand2) => {
   const len1 = strand1.length;
   const len2 = strand2.length;
```

Comment:
Now that len1 and len2 are only used once, I think declaring these extra variables is not necessary anymore. The new line 5 could use `.length` directly.

Reply:
I did this change myself so this can be merged.

Reply:
Yes, sorry I just saw that, thanks :ok_hand: